src/fastly/include-util.js

```diff
@@ -44,7 +45,7 @@ function injectConsts(content, constants) {
     .entries(constants)
     .filter(([key, value]) => !!key && !!value)
     .reduce((str, [key, value]) => str
-      .replace(`{"const:${key}"}`, `"${value}"`),
+      .replace(new RegExp(escapeStringRegexp(`{"const:${key}"}`), 'g'), `"${value}"`),
```

Comment:
need to use regexp replace  with global flag, since nodejs doesn't have a `String.replaceAll()`